deploy/roles/spark/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+spark_tar_file:             "spark-1.3.1-bin-hadoop2.6.tgz"
```

Comment:
@olivermel This still needs to be the correct version, 1.5.2, this is the link you should use: http://mirror.cc.columbia.edu/pub/software/apache/spark/spark-1.5.2/spark-1.5.2-bin-hadoop2.6.tgz